test/system/multifactor_auths_test.rb

```diff
@@ -87,8 +87,9 @@ def redirect_test_mfa_disabled(path)
     sign_in
     visit path
 
-    assert(page.has_content?("Enabling multi-factor auth"), "#{path} was not redirected to mfa setup page")
+    assert(page.has_content?("you are required to set up multi-factor authentication"))
```

Comment:
nit: Originally this was testing if the user was redirected to the set up MFA page, and it was changed to test the warning which is good to test. Could we still have an assertion to test whether we're on the right page so it's clear what page the user is supposed to be on? (ie. `assert_current_path(edit_settings_path)`).

Reply:
Done!